src/main/java/utility/Settings.java

```diff
@@ -43,8 +46,12 @@
     public static final double JUMP_SPEED_WALKER = 3 * MONSTER_SPEED;
     public static final double JUMP_HEIGHT_WALKER = 200;
 
+    public static String[] names = new String[2];
```

Comment:
Checkstyle says that names should be private and have an accessor method. 
<bug